### ISSUE TITLE
Detection of hardware encoder is incorrect

### ISSUE THREAD

**Issue by mm318** [aws-robotics/kinesisvideo-encoder-common#36]
With https://github.com/aws-robotics/kinesisvideo-encoder-common/pull/35, there is an initial attempt to open the `h264_omx` encoder to detect whether or not it is available. However, the attempt to open it gives a message like:
```
[h264_video_encoder-11] [h264_omx @ 0xea8260] The encoder timebase is not set.
```
and fails to correctly identify that the encoder is actually there available and working.

I suspect this is because the `h264_omx` encoder is being attempted to be opened in `is_codec_available()` with unset parameters (where the timebase, or framerate, is not set as the error message above indicates).

**Comment by ryanewel** [comment]
A fix has been merged into master. Leaving this issue open until the release is available in apt

**Comment by mm318** [comment]
The fix has been released to version 2.0.3.
